apps/documents/models.py

```diff
@@ -85,3 +110,32 @@ def shipper_wallet_id(self):
     @property
     def vault_id(self):
         return f"{self.shipment.vault_id}"
+
+
+class CsvDocument(models.Model):
+    id = models.CharField(primary_key=True, default=random_id, max_length=36)
+    name = models.CharField(max_length=36, null=False, blank=False)
+    storage_credentials_id = models.CharField(null=False, max_length=36)
+    shipper_wallet_id = models.CharField(null=False, max_length=36)
+    carrier_wallet_id = models.CharField(null=False, max_length=36)
+    owner_id = models.CharField(null=False, max_length=36)
+    updated_by = models.CharField(null=False, max_length=36)
```

Comment:
I think we can get rid of `owner_id` but  `updated_by` is necessary since it is the user who upload and will be needed later in lambda in order to masquerade